components/src/core/components/Input/Time/TimeInput.vue

```diff
@@ -13,9 +13,13 @@
         tabindex="0"
         maxlength="5"
       />
-      <div class="oxd-time-input-am-pm-wrapper" :class="amPmWrapperClasses">
+      <div
+        class="oxd-time-input-am-pm-wrapper"
+        :class="amPmWrapperClasses"
```

Comment:
Let's move this to `amPmWrapperClasses` as well

```
const amPmWrapperClasses = computed(() => {
  return {
    'oxd-time-input-am-pm-wrappe' : true,
    '--disabled': props.disabled,
  };
});
```

Reply:
this is done